docs/4. Product Features/03. Flows/4. Others/01. Flows not working - Troubleshoot checklist.md

```
@@ -69,6 +77,19 @@ Also, make sure, that the default optin message from the Gupshup portal is disab
 
 <img width="605" alt="image" src="https://github.com/glific/docs/assets/122982753/1393af27-b069-4d2c-b80f-61cbb6d45b7a"/>
 
+13. `Infinite loop detected` - The system monitors nodes (node_uuid) for repeated triggers and message bodies (body). Infinite loops are detected using two thresholds:
+
+   -  Same Node, Different Messages
+
+      > - **Condition:** A node is triggered multiple times with different message bodies.
+      > - **Threshold:** 5 times
+      > - **Action:** After the 5th different message hits the same node, the system flags it as an **infinite loop**.
+
+  -  Same Node, Same Message
```

markdownlint-cli2 (0.20.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 3; Actual: 2

(MD005, list-indent)